src/o365/spo/commands/listitem/listitem-record-declare.ts

```diff
@@ -0,0 +1,309 @@
+import auth from "../../SpoAuth";
+import config from "../../../../config";
+import commands from "../../commands";
+import GlobalOptions from "../../../../GlobalOptions";
+import * as request from "request-promise-native";
+import {
+  CommandOption,
+  CommandValidate,
+  CommandTypes,
+  CommandError
+} from "../../../../Command";
+import SpoCommand from "../../SpoCommand";
+import Utils from "../../../../Utils";
+import { Auth } from "../../../../Auth";
+import {
+  ContextInfo,
+  ClientSvcResponse,
+  ClientSvcResponseContents,
+} from "../../spo";
+import { ClientSvc, IdentityResponse } from "../../common/ClientSvc";
+
+const vorpal: Vorpal = require("../../../../vorpal-init");
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  id: string;
+  date?: string;
+}
+
+class SpoListItemDeclareRecord extends SpoCommand {
```

Comment:
Following the naming convention, the class should be named `SpoListItemRecordDeclareCommand`

Reply:
Done!